.travis.yml

```diff
@@ -1,20 +1,46 @@
 sudo: no
 language: python
-env:
-  matrix:
-    - TOXENV=docs
-    - TOXENV=py27-1.6
-    - TOXENV=py27-1.7
-  global:
-    - PIP_WHEEL_DIR=$HOME/.wheels
-    - PIP_FIND_LINKS=file://$PIP_WHEEL_DIR
-cache:
-  directories:
-    - $PIP_WHEEL_DIR
+cache: pip
+matrix:
+  include:
+    - python: "2.7"
```

Comment:
Interesting. I may need help with how to do that with the sliding Python / Django support pairs.